### DISCUSSION TITLE
Unable to run lotus-miner net/identify   go-log@v1.0.4/log.go:175   error reading identify message: stream reset

### DISCUSSION THREAD

**Discussion by wzg8723** [filecoin-project/lotus#5653] [General Q&A] [answered]
Hi there ,

I 've got the error message below when I run the lotus-miner

Could you please give me some hints?
Thanks

![image](https://user-images.githubusercontent.com/30481496/108041848-f39fbc00-6ff3-11eb-9a1d-99dae97baeb9.png)

**Comment by seantw1974** [comment]
I have same issue ..
Please help!!
How to resolve ?

**Comment by Meest-NAM** [comment]
same 

help....plz

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
first of all: its a warning message, not an error.

the connection got interrupted for some reason and the stream got reset. shouldn't be a bigger problem. restart daemon and miner and see if the problems are persistent
